lib/UnoCore/Source/Uno/IO/Directory.uno

```diff
@@ -273,7 +273,7 @@ namespace Uno.IO
             @}
             else if defined(CPLUSPLUS)
             @{
-                chdir(uCString($0).Ptr);
+                int res = chdir(uCString($0).Ptr);
```

Comment:
I think it's cleaner to just do `(void)chdir(uCString($0).Ptr);`

Reply:
This will avoid an unused variable, so let's do that instead. Thanks!

Reply:
Casting to `void` doesn't silence the warning... :/

https://travis-ci.org/fuse-open/uno/jobs/527629788#L2097